doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1037 @@
+    * :c:func:`nrf_cloud_pgps_process_update` function that stores a portion of a P-GPS download to flash.
+    * :c:func:`nrf_cloud_pgps_finish_update` function that a user of the P-GPS library calls when the custom download completes.
+
+  * Reduced logging level for many messages to debug (``DBG``).
+
+* :ref:`lib_azure_iot_hub` library:
+
+  * Reworked to use `Azure SDK for Embedded C`_.
+  * Updated the APIs for both IoT Hub and DPS interaction.
+    The applications and samples that use the library have been updated accordingly.
+
+* :ref:`lib_download_client` library:
+
+  * Fixed:
+
+    * Handling of duplicated CoAP packets.
+    * Handling of timeout errors when using CoAP.
+
+Other libraries
+---------------
+
+* Added:
+
+  * Documentation for the :ref:`lib_adp536x` library.
+  * Documentation for the :ref:`lib_flash_map_pm` library.
+
+* :ref:`lib_flash_patch` library:
+
+  * Added documentation page.
+  * Modified the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option, so that it can be used on the nRF52833 SoC.
+
+* :ref:`doc_fw_info` module:
+
+  * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
+
+* :ref:`nrf_rpc_ipc_readme`:
+
+  * This is a new library.
+    It can use different transport implementation for each nRF RPC group.
+  * Memory for remote procedure calls is now allocated on a heap instead of the calling thread stack.
+
+* :ref:`emds_readme`:
+
+  * Updated :c:func:`emds_entry_add` to no longer use heap, but instead require a pointer to the dynamic entry structure :c:struct:`emds_dynamic_entry`.
+    The dynamic entry structure should be allocated in advance.
+
+* :ref:`mod_memfault`:
+
+  * Added default metrics for Bluetooth.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* :ref:`caf_ble_adv`:
+
+  * Added :kconfig:option:`CONFIG_CAF_BLE_ADV_FILTER_ACCEPT_LIST` Kconfig option.
+    The option is used instead of :kconfig:option:`CONFIG_BT_FILTER_ACCEPT_LIST` option to enable the filter accept list.
+  * Integrated :ref:`bt_le_adv_prov_readme`.
+    The subsystem is now used instead of the :file:`*.def` file to configure advertising data and scan response data.
+  * Bluetooth device name is no longer automatically included in scan response data.
+    A dedicated data provider (:kconfig:option:`CONFIG_BT_ADV_PROV_DEVICE_NAME`) can be used to add the Bluetooth device name to the scan response data.
+  * Added :c:struct:`ble_adv_data_update_event` that can be used to trigger update of advertising data and scan response data during undirected advertising.
+    When the event is received, the module gets new data from providers and updates advertising payload.
+  * Added a wakeup call when connection is made in the grace period.
+    With this change, the call wakes up the whole system to avoid inconsistent power state between modules.
```

Comment:
```suggestion
  * Added:
    *  :kconfig:option:`CONFIG_CAF_BLE_ADV_FILTER_ACCEPT_LIST` Kconfig option.
      The option is used instead of :kconfig:option:`CONFIG_BT_FILTER_ACCEPT_LIST` option to enable the filter accept list.
      * :c:struct:`ble_adv_data_update_event` that can be used to trigger update of advertising data and scan response data during undirected advertising.
      When the event is received, the module gets new data from providers and updates advertising payload.
    * A wakeup call when connection is made in the grace period.
      With this change, the call wakes up the whole system to avoid inconsistent power state between modules.

  * Integrated :ref:`bt_le_adv_prov_readme`.
    The subsystem is now used instead of the :file:`*.def` file to configure advertising data and scan response data.
  * Bluetooth device name is no longer automatically included in scan response data.
    A dedicated data provider (:kconfig:option:`CONFIG_BT_ADV_PROV_DEVICE_NAME`) can be used to add the Bluetooth device name to the scan response data.
  
```